### PR TITLE
tracking-issue: Support org-wide tracking issues

### PR DESCRIPTION
**Pull request by tsenart**
This commit extends `tracking-issue` with support for issues from all
repositories of a GitHub organization.

Issues from private repos are represented with the name of the
repository and the issue number instead of the title.

Additionally, this changes the emoji rendering to support multiple
categories if defined.

### CODE REVIEW

File: internal/cmd/tracking-issue/main.go [hunk 1]
```diff
@@ -145,41 +185,16 @@ func assignee(user *github.User) string {
...
-
-func listIssues(ctx context.Context, cli *github.Client, milestone int, labels ...string) (issues []*github.Issue, _ error) {
-	opt := &github.IssueListByRepoOptions{
-		Milestone:   strconv.Itoa(milestone),
-		Labels:      labels,
+func listIssues(ctx context.Context, cli *github.Client, org string, labels ...string) (issues []*github.Issue, _ error) {
+	opt := &github.IssueListOptions{
+		Filter:      "all",
 		State:       "all",
+		Labels:      labels,
```
Comment (keegancsmith) [inline internal/cmd/tracking-issue/main.go]: out of interest how much slower is this now? Having the Milestone specified bounded the response size.

Comment (tsenart) [reply internal/cmd/tracking-issue/main.go]: It's much slower. It takes like 10 seconds for our 3.13 tracking issue.

Unfortunately you can't filter by milestone when listing org issues, and using the GraphQL API was a deep rabbit hole which I briefly got into, but quickly left. :-)

Comment (nicksnyder) [reply internal/cmd/tracking-issue/main.go]: I don't know what this Go library is capable of, but it is easy to do what you want by directly calling the HTTP API:

```
# I assume you will get private issues from this if you specify a token, but worth verifying
curl 'https://api.github.com/search/issues?q=org:sourcegraph+milestone:3.13' -H "Authorization: token $GITHUB_TOKEN"
```

Comment (tsenart) [reply internal/cmd/tracking-issue/main.go]: Ah, good idea @nicksnyder. I hadn't thought of using the search API.

Comment (tsenart) [reply internal/cmd/tracking-issue/main.go]: Essentially, this library is using the REST API, and this endpoint in particular right now: [`GET /orgs/:org/issues`](https://developer.github.com/v3/issues/#list-issues)

Comment (tsenart) [reply internal/cmd/tracking-issue/main.go]: OK, so I tried using the search API for this, and unfortunately it doesn't return the repository object along with the issue that belongs to it, which we need to determine if the repo is private or not.

Comment (tsenart) [reply internal/cmd/tracking-issue/main.go]: Alright, back to the GraphQL rabbit hole and I think I got this working:

```graphql
query {
  search(
    first: 100
    type: ISSUE
    query: "org:sourcegraph label:team/core-services milestone:3.13"
  ) {
    pageInfo {
      endCursor
      hasNextPage
    }
    nodes {
      ... on Issue {
        title
        body
        repository {
          isPrivate
        }
        labels(first: 25) {
          nodes {
            name
          }
        }
      }
    }
  }
}
```

Comment (keegancsmith) [reply internal/cmd/tracking-issue/main.go]: I believe for GraphQL + GitHub people have converged on https://github.com/shurcooL/githubv4

Comment (tsenart) [reply internal/cmd/tracking-issue/main.go]: Addressed in #8018

**Pull request merged**